addons/website_slides/tests/test_slide_channel.py

```diff
@@ -134,6 +135,21 @@ def test_mail_completed_with_different_templates(self):
             ['Congratulations! You completed %s' % self.channel.name, 'ATestSubject']
         )
 
+    def test_share_without_template(self):
+        channel_without_template = self.env['slide.channel'].create({
+            'name': 'Cours Without Channel 2',
+            'slide_ids': [(0, 0, {
+                'name': 'Test Slide 2'
+            })],
+            'share_channel_template_id': False
+        })
+        all_channels = self.channel | channel_without_template
+        with self.assertRaises(UserError) as cm:
+            all_channels.sudo()._send_share_email("test@test.com")
```

Comment:
Why `sudo` here? Could you use an authorized user instead, like 'user_officer' (see other examples in the file)?